tests/vppoutputencode.cpp

```diff
@@ -98,6 +103,17 @@ static void setEncodeParam(const SharedPtr<IVideoEncoder>& encoder,
     encoder->getParameters(VideoParamsTypeAVC, &encVideoParamsAVC);
     encVideoParamsAVC.idrInterval = encParam->idrInterval;
     encVideoParamsAVC.size = sizeof(VideoParamsAVC);
+#if YAMI_CHECK_API_VERSION(0, 2, 1)
```

Comment:
Yeah, seems it different from libva. Both VAAPI interface and test case are put in libva project. It is a tough thing, let's keep it right now.